### ISSUE TITLE
Text cursor is in wrong location when selecting after last character in a variable

### ISSUE THREAD

**Issue by Michael-Grover** [accordproject/web-components#179]
# Bug Report 🐛
When a user selects after the last character in a variable, the text cursor appears to be slightly outside of the variable, instead of where the text will appear when the user types a character.

## Expected Behavior
Text cursor should be where the text will appear, like this:

![image](https://user-images.githubusercontent.com/40569793/91342025-6ef08580-e7a8-11ea-9a57-64f279d72836.png)
![image](https://user-images.githubusercontent.com/40569793/91342171-a4956e80-e7a8-11ea-90ec-1758212d9678.png)

## Current Behavior
![image](https://user-images.githubusercontent.com/40569793/91341961-584a2e80-e7a8-11ea-9c93-d49ff706da16.png)

**Comment by khanjasir90** [comment]
@Michael-Grover can i work on this!

**Comment by Michael-Grover** [comment]
Hi @khanjasir90 yes, thanks! @DianaLease has assigned the task to you.

**Comment by irmerk** [comment]
@Michael-Grover I don't believe this is an issue. When looking at our [current code](https://ap-web-components.netlify.app/?path=/story/contract-editor--contract-editor), it is already the way you describe the solution.

**Comment by Michael-Grover** [comment]
I tested the bug and it was still happening on my computer, but then I updated Google Chrome and now the text cursor is showing up in the correct place 👍
